infra/docker/api/Dockerfile

```diff
@@ -21,6 +24,7 @@ COPY ./apps/api ./apps/api
 COPY ./packages ./packages
 # TODO: follow up pr to remove dependencies on web
 COPY ./apps/web ./apps/web
+COPY ./tests ./tests
```

Comment:
@jalada what was the need to copy tests for? Wondering if we can find a way to not include these in the image.

Reply:
Ah I see. We have the test file nested in the API. 

Ok, I will add a follow-up item for us to restructure the file system a bit to not need this. Thanks!